lib/testmap.py

```diff
@@ -21,211 +21,215 @@
 
 from lib.constants import TEST_OS_DEFAULT
 
-COCKPIT_SCENARIOS = {'networking', 'storage', 'expensive', 'other'}
```

Comment:
All of these changes (probably? maybe there are some changes in between that I missed) are just lints